### PR TITLE
Fix references for distinctCountHLLMv and distinctCountBitmapMv

### PR DESCRIPTION
**Pull request by deemoliu**
Fix the content of the following pinot-docs
configuration-reference/functions/distinctcountbitmapmv.md: the content is pointing to distinctcounthllmv
configuration-reference/functions/distinctcounthllmv.md: the content is pointing to distinctcountbitmapmv.md

### CODE REVIEW

**Comment by deemoliu** [comment]
cc: @Jackie-Jiang

**Comment by Jackie-Jiang** [comment]
cc @kelseiv

**Review by Jackie-Jiang — APPROVED** [review]

File: configuration-reference/functions/distinctcountbitmapmv.md [hunk 1]
```diff
@@ -1,27 +1,37 @@
...
-  This section contains reference documentation for the DISTINCTCOUNTHLLMV
-  function.
+This section contains reference documentation for the DISTINCTCOUNTBITMAPMV
+function.
 ---
 
 # DISTINCTCOUNTBITMAPMV
 
-Returns an approximate distinct count using HyperLogLog in a group
+Returns the count of distinct row values in a group. This function is accurate for INT or dictionary encoded column, but approximate for other cases where hash codes are used in distinct counting and there may be hash collision.
```
Comment (kelseiv) [inline configuration-reference/functions/distinctcountbitmapmv.md]:
```suggestion
Returns the count of distinct row values in a group. This function is accurate for an INT or dictionary encoded column, but approximate for other cases where hash codes are used in distinct counting and there may be hash collision.
```

File: configuration-reference/functions/distinctcounthllmv.md [hunk 2]
```diff
@@ -1,37 +1,27 @@
...
-  This section contains reference documentation for the DISTINCTCOUNTBITMAPMV
-  function.
+This section contains reference documentation for the DISTINCTCOUNTHLLMV
+function.
 ---
 
 # DISTINCTCOUNTHLLMV
 
-Returns the count of distinct row values in a group. This function is accurate for INT or dictionary encoded column, but approximate for other cases where hash codes are used in distinct counting and there may be hash collision.
+Returns an approximate distinct count using HyperLogLog in a group
```
Comment (kelseiv) [inline configuration-reference/functions/distinctcounthllmv.md]:
```suggestion
Returns an approximate distinct count using HyperLogLog in a group.
```

**Review by kelseiv — APPROVED** [review]
Nice catch, thank you, @deemoliu @Jackie-Jiang

**Pull request merged**